apis/r/configure

```diff
@@ -10,8 +10,14 @@ if test -z "${R_HOME}"; then
 fi
 
 ### for single-cell-data/apis/r repo, consider completing sources
-if test -f src/lib/libtiledbsoma.a; then
-    echo "** static library present"
+pkg-config --exists tiledbsoma
+if [ $? -eq 0 ]; then
+    lib=`pkg-config --libs-only-L tiledbsoma` | cut -d'_' -f 2
+    export LD_LIBRARY_PATH=$lib:$LD_LIBRARY_PATH
+    sudo ldconfig
+    echo "** system installed static library present"
+elif test -f src/lib/libtiledbsoma.a; then 
+    echo "** static library present in src/lib"
```

Comment:
This second block is what is currently used. I think it should remain the default.